codalab/lib/server_util.py

```diff
@@ -7,10 +7,39 @@
 import sys
 import urllib
 
-from bottle import request, HTTPResponse, redirect
+from bottle import abort, request, HTTPResponse, redirect, app, local
 from oauthlib.common import to_unicode, bytes_type
 
 
+def query_get_bool(key, default=False):
+    value = request.query.get(key, None)
```

Comment:
Why does pack_params convert bools to int (i.e. 0 and 1), but this checks for True?